base_attachment_object_storage/models/ir_attachment.py

```diff
@@ -228,9 +265,9 @@ def do_in_new_env(self, new_cr=False):
                 with closing(registry.cursor()) as cr:
                     try:
                         yield self.env(cr=cr)
-                    except:
-                        cr.rollback()
-                        raise
+                    except Exception:
+                        _logger.exception("Error creating new odoo "
+                                          "environment")
```

Comment:
I really doubt that you want to swallow the exception here. Also, there is no rollback.

For the record, the code in 13.0 is:

 https://github.com/camptocamp/odoo-cloud-platform/blob/af8cee48fb6b9a8a6acf924d40409cc693a2aa0c/base_attachment_object_storage/models/ir_attachment.py#L253-L255

In the 13.0 code, no logging is needed because the exception is re-raised, the goal of the exception handler is only to rollback the transaction.

Reply:
I understand you point. Doing the revert.